hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/configuration/FlinkOptions.java

```diff
@@ -746,11 +760,11 @@ public static Configuration flatOptions(Configuration conf) {
           : key;
       propsMap.put(subKey, value);
     });
-    return fromMap(propsMap);
+    return Configuration.fromMap(propsMap);
   }
```

Comment:
Revert all the usage of `Configuration.fromMap`

Reply:
the same code does not need to be repeated.

Reply:
WDYT? :)